i18n/language/language.go

```diff
@@ -17,23 +19,35 @@ type Language struct {
 	*PluralSpec
 }
 
+// NewLanguage returns new Language with specified tag and spec.
+func NewLanguage(tag string, spec *PluralSpec) *Language {
```

Comment:
If this is a problem outside of the package, then this whole PR is backwards incompatible.